samples/grids/hierarchical-grid/data-performance-virtualization/wwwroot/events.js

```diff
@@ -0,0 +1,54 @@
+igRegisterScript("OnGridRendered", () => {
+    const grid = document.getElementById("hGrid");
+
+    grid.isLoading = true;
+
+    getData({ parentID: null, rootLevel: true, key: "Customers" }).then(
+        (data) => {
+            grid.isLoading = false;
+            grid.data = data;
+            grid.markForCheck();
+        });
+}, false)
+
+igRegisterScript("OnGridCreated", (args) => {
+    const context = args.detail;
+    const _parentKey = context.owner.childDataKey === "Orders" ? "CustomerID" : "OrderID";
+    const dataState = {
+        key: context.owner.childDataKey,
+        parentID: context.parentID,
+        parentKey: _parentKey,
+        rootLevel: false,
+    };
+
+    context.grid.isLoading = true;
+
+    getData(dataState).then((data) => {
+        context.grid.isLoading = false;
+        context.grid.data = data;
+        context.grid.markForCheck();
+    });
+}, false)
+
+const DATA_URL = `https://services.odata.org/V4/Northwind/Northwind.svc/`;
```

Comment:
Similar to the case in [this PR for WC](https://github.com/IgniteUI/igniteui-wc-examples/pull/683#pullrequestreview-1979913935), the URL is broken and no data is loaded.